.ebextensions/enable_mod_deflate.conf

```diff
@@ -23,10 +23,14 @@ BrowserMatch ^Mozilla/4\.0[678] no-gzip
 
 # MSIE masquerades as Netscape, but it is fine
 BrowserMatch \bMSI[E] !no-gzip !gzip-only-text/html
+</IfModule>
 
 <IfModule mod_headers.c>
 # Make sure proxies don't deliver the wrong content
 Header append Vary User-Agent env=!dont-vary
 </IfModule>
 
-</IfModule>
```

Comment:
Ideally this should be in a seperate PR as discussed.

Reply:
just the fix for incorrect nesting left